playground/README.md

```diff
@@ -16,6 +16,7 @@ To build the playground locally, run these commands from this directory:
 ```bash
 # Build the playground crate to WebAssembly:
 # (The wasm-opt step can be replaced with a simple file copy.)
+rustup target add wasm32-unknown-unknown
```

Comment:
Because the user does not need to run this every build, and will not necessarily be using `rustup`, I would prefer to put it in a separate code block, as well as extend the existing description of additional dependencies to explain what it's for.